### DISCUSSION TITLE
PrimeBlocks license

### DISCUSSION THREAD

**Discussion by dalenguyen** [primefaces/community#267] [PrimeNG] [answered]
I saw the license in the PrimeBlocks mentioning that:

```
Can I use PrimeBlocks in open-source projects?
Due to the license, it is not possible to use the blocks in an open source project where code is publicly available.
```

Does it mean all blocks cannot be used in opensource project?

How about FREE blocks that are available on the website?

<img width="1069" alt="image" src="https://github.com/primefaces/community/assets/14116156/dae2ebf3-80b3-4bd8-a6eb-7394bc7831bd">

Can it be used anywhere open-source / commercial...?

**Comment by cetincakiroglu** [comment, marked as answer]
Due to the licence you cannot use it in open source projects since it means distribution of the code. You can use free ones in your open-source project.

**Reply by dalenguyen** [reply]
Thanks for your answer 😄

**Reply by rawkode** [reply]
Any plans to change this? Love the library, want to buy; but my project is open source.
